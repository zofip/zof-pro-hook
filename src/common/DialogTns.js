import React from 'react';

import Button from '@material-ui/core/Button';
import Dialog from '@material-ui/core/Dialog';
import DialogActions from '@material-ui/core/DialogActions';
import DialogContent from '@material-ui/core/DialogContent';
import DialogContentText from '@material-ui/core/DialogContentText';
import DialogTitle from '@material-ui/core/DialogTitle';


const DialogTns = ({ title, message, open, handleClose }) => {

    return (
        <React.Fragment>
            <Dialog
                fullWidth
                maxWidth='xs'
                open={open}
                onClose={handleClose}
                aria-labelledby="max-width-dialog-title">
                <DialogTitle id="max-width-dialog-title">{title}</DialogTitle>
                <DialogContent>
                    <DialogContentText>
                        {message}
                    </DialogContentText>
                </DialogContent>
                <DialogActions>
                    <Button onClick={handleClose} color="primary">
                        Close
                    </Button>
                </DialogActions>
            </Dialog>
        </React.Fragment>
    );
}

export default DialogTns;
