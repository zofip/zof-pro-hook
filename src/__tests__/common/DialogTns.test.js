import React from 'react';
import { shallow } from 'enzyme';
import DialogTns from '../../common/DialogTns';

describe('<DialogTns />', () => {
   it('Rendering', () => {
      const dialogTns = shallow(<DialogTns />);
      expect(dialogTns).toMatchSnapshot();
   });
});