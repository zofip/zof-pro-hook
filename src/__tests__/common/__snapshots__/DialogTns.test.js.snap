// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DialogTns /> Rendering 1`] = `
<Fragment>
  <WithStyles(ForwardRef(Dialog))
    aria-labelledby="max-width-dialog-title"
    fullWidth={true}
    maxWidth="xs"
  >
    <WithStyles(ForwardRef(DialogTitle))
      id="max-width-dialog-title"
    />
    <WithStyles(ForwardRef(DialogContent))>
      <WithStyles(ForwardRef(DialogContentText)) />
    </WithStyles(ForwardRef(DialogContent))>
    <WithStyles(ForwardRef(DialogActions))>
      <WithStyles(ForwardRef(Button))
        color="primary"
      >
        Close
      </WithStyles(ForwardRef(Button))>
    </WithStyles(ForwardRef(DialogActions))>
  </WithStyles(ForwardRef(Dialog))>
</Fragment>
`;
